sdk/ios/Configsum/Configsum/Configsum/Inputs/Context.swift

```diff
@@ -42,7 +44,7 @@ public class Context: Codable {
     
     private struct Location: Codable {
         let locale: String
-        let timezoneOffset: String
+        let timezoneOffset: Int
```

Comment:
what about countries with half hour timezones?

Reply:
They will be reflected in the seconds as well just with an offset of `1800` instead of `3600`.